serve/overview.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Serve results: access and use your processed data"
+description: "RisingWave allows you to access and use insights from your streaming data immediately. It also functions like any other database, allowing you to query batch or raw data that you've inserted."
+---
+
+This section explains how to query and interact with data in RisingWave.
+
+RisingWave offers several methods for serving results, catering to various use cases, from ad-hoc analysis to application integration.
+
+## Query with `SELECT` statements
+
+Retrieve data directly from RisingWave using standard SQL SELECT queries against tables or materialized views. Use this method for ad-hoc analysis, exploring the latest results, and extracting specific data subsets.
+
+For syntax details, see [`SELECT`](/sql/commands/sql-select).
```

Comment:
Can add link to the ad-hoc processing here https://github.com/risingwavelabs/risingwave-docs/blob/main/processing/overview.mdx#ad-hoc-on-read-vs-streaming-on-write